### DISCUSSION TITLE
can't get H and V accuracy with antenna direct connexion (no mock location)

### DISCUSSION THREAD

**Discussion by ewaca** [opengisch/QField#3512] [Q&A]
Hi,

I'm using an ardusimple F9p with 2 BT modules.
1 on UART1 receiving caster's datas from an ntrip client
1 on UART2 to send corrected position by F9P to qfield
My configration is : 

Port UART1 
IN : UBX + NMEA + RTCM3 
OUT : UBX + NMEA + RTCM3 
messages : UBX-RXM RAWX, UBX-RXM SFRBX GGA, GLL, GSA, GSV, RMC, VTG, ZDA 

Port UART2 : 
IN : RTCM3 
OUT : NMEA 
messages : GGA, GLL, GSA,  GSV, RMC, VTG, ZDA 

when i active "actual position information" i can see data in HDOP and VDOP fields, but nothing in H and V accuracy.
I have tried to activate PVT messages, it works through usb in u-center but it does not work throught UART1/2 in Qfield.

Is there a message that I forgot to get the H/V accuracy ?

**Comment by bettellam** [comment]
@ewaca Add NMEA GST and you are solved !!!
Good work.
